### DISCUSSION TITLE
Assistance for Deleting Inactive users

### DISCUSSION THREAD

**Discussion by Venkatesh-Dayani23** [Sunbird-Lern/Community#180] [Issues]
Hi Team,

We are currently using Sunbird version 5.0.0 and require assistance in the process to **find out the inactive users from more than a year** and **want to delete the inactive users**.Please provide us with instructions for this procedure.

cc: @raghupathiguduri

**Comment by AmiableAnil** [comment]
This is the design doc we have for delete user https://project-sunbird.atlassian.net/wiki/spaces/SBDES/pages/3354492949/Design+Delete+Account+Functionality
Check this if it is useful. We have implemetented the delete user functionality as a part of release-7.0.0

**Reply by Venkatesh-Dayani23** [reply]
Thanks for the update @AmiableAnil

**Reply by raghupathiguduri** [reply]
@AmiableAnil Is there any way that we can delete the users from the system without upgrading the version to 7.0.0 ? If possible, how we can do that ?
